app/workers/evss/disability_compensation_form/submit_form0781.rb

```diff
@@ -31,6 +31,21 @@ class SubmitForm0781 < Job
         Metrics.new(STATSD_KEY_PREFIX, msg['jid']).increment_exhausted
       end
 
+      def get_docs(submission_id, uuid)
+        @submission_id = submission_id
```

Comment:
What is the purpose of setting this instance variable? Does it get used somewhere else?

Reply:
Yea it does get used but in a sort of round-about way. 

Other places further down (for example line 36) call a variable (or a method rather, but that isnt obvious from how the code is) `submission`

That is a method in the `Job` class that this class is inheriting from. https://github.com/department-of-veterans-affairs/vets-api/blob/25d9dd9ce257beb263514ef5884291af44e812c7/app/workers/evss/disability_compensation_form/job.rb#L22

This variable is usually set when the perform is called, but we are not calling that here, so I need to set it before running the rest of my method so that the class has reference to the claim it needs to parse.